samples/buildSrc/src/main/kotlin/RunKotlinNativeTask.kt

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2010-2018 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license
+ * that can be found in the license/LICENSE.txt file.
+ */
+
+import groovy.lang.Closure
+import org.gradle.api.DefaultTask
+import org.gradle.api.Task
+import org.gradle.api.tasks.TaskAction
+import org.jetbrains.kotlin.gradle.plugin.KotlinTarget
+import javax.inject.Inject
+
+open class RunKotlinNativeTask @Inject constructor(
+        private val myTarget: KotlinTarget
+): DefaultTask() {
+
+    var buildType = "RELEASE"
+    var workingDir: Any = project.projectDir
```

Comment:
`Any`?

Reply:
Yep. Because we want to accept String, File, Provider, etc - anything that Gradle task is capable to eat.